tasks/src/text-to-image/data.ts

```diff
@@ -0,0 +1,34 @@
+import type { TaskDataCustom } from "../Types";
+
+const taskData: TaskDataCustom = {
+    datasets:      [
+        {
+            description: "A large-scale image dataset containing images of everyday objects and people with their textual descriptions.",
+            id:          "ChristophSchuhmann/MS_COCO_2017_URL_TEXT",
+        },
+        {
+            description: "A dataset created from Midjourney User Prompts & Generated Images",
+            id:          "nateraw/midjourney-texttoimage-new",
+        },
+    ],
+    demo:          {
+        inputs:  [],
+        outputs: [],
+    },
+    metrics:       [],
+    models:        [
+        {
+            description: "A model that can be used to generate images based on text prompts. The DALL·E Mega model is the largest version of DALLE Mini.",
```

Comment:
I'm not sure the DALL-E Mega sentence is super relevant for the audience. The first sentence is :fire:

Reply:
I thought it would be helpful for someone who wants to use dall-e mini to see that dall-e mega is the biggest version in the description field. Thank you for your feedback.

Reply:
The datasets I've mentioned are more usable (they're canonical and their code is implemented so they're loaded easier and better integrated with `datasets` library) that's why I recommended them. 🙂 You can ask for a review again if you make a change.